### ISSUE TITLE
Set up remaining pages

### ISSUE THREAD

**Issue by loukkevin** [loukkevin/EventPlatform#1]
We need to create a subscriptions page as well as the profile page.

**Comment by tghislain** [comment]
Can we meet tomorrow at 3?
I waa able to set up a database and some code to access too

**Comment by loukkevin** [comment]
Yeah I am done with class at 3. We should meet as much as possible to get as much done as we can. We still have to do the video presentation thing as well.
